src/main/java/org/broadinstitute/hellbender/tools/funcotator/FilterFuncotations.java

```diff
@@ -0,0 +1,334 @@
+
+    public String getFilterName() {
+        return filterName;
+    }
+}
+
+
+class ClinVarFilter extends FuncotationFilter {
+
+    private static final String ACMG_DISEASE_FUNCOTATION = "ACMG_recommendation_Disease_Name";
+    private static final String CLIN_VAR_VCF_CLNSIG = "ClinVar_VCF_CLNSIG";
+
+    ClinVarFilter() {
+        super("CLINVAR");
+    }
+
+    @Override
+    List<FuncotationFiltrationRule> getRules() {
+        final List<FuncotationFiltrationRule> clinVarFiltrationRules = new ArrayList<>();
+
+        // 1) The gene name must be on the ACMG59 list (American College of Medical Genomics).
+        clinVarFiltrationRules.add(new FuncotationFiltrationRule("ClinVar-ACMG59") {
+            @Override
+            boolean applyRule(final VariantContext variant, final Map<String, String> prunedTranscriptFuncotations) {
+                return prunedTranscriptFuncotations.containsKey(ACMG_DISEASE_FUNCOTATION);
+            }
+        });
+
+        // 2) ClinVar annotations specifies Pathogenicity or Likely pathogenic.
+        clinVarFiltrationRules.add(new FuncotationFiltrationRule("ClinVar-pathogenic") {
+            @Override
+            boolean applyRule(final VariantContext variant, final Map<String, String> prunedTranscriptFuncotations) {
+                final String significance = prunedTranscriptFuncotations.getOrDefault(CLIN_VAR_VCF_CLNSIG, "");
+                return significance.contains("Pathogenic") || significance.contains("Likely_pathogenic");
+            }
+        });
+
+        // 3) Frequency: Max Minor Allele Freq is ≤5% in GnoMAD (ExAC for Proof of Concept)
+        clinVarFiltrationRules.add(new FuncotationFiltrationRule("ClinVar-MAF") {
+            @Override
+            boolean applyRule(final VariantContext variant, final Map<String, String> prunedTranscriptFuncotations) {
+                return getMaxMinorAlleleFreq(prunedTranscriptFuncotations) <= 0.05;
+            }
+        });
+        return clinVarFiltrationRules;
+    }
+}
+
+class LofFilter extends FuncotationFilter {
+
+    private static final String LOF_GENE_FUNCOTATION = "ACMGLMMLof_LOF_Mechanism";
+    private static final String FRAME_SHIFT_PREFIX = "FRAME_SHIFT_";
+    private static final List<String> CONSTANT_LOF_CLASSIFICATIONS = Arrays.asList("NONSENSE", "START_CODON_DEL", "SPLICE_SITE");
+
+    private final String classificationFuncotation;
+
+    LofFilter(final FilterFuncotations.ReferenceVersion ref) {
+        super("LOF");
+        this.classificationFuncotation = "Gencode_" + ref.gencodeVersion + "_variantClassification";
+    }
+
+    @Override
+    List<FuncotationFiltrationRule> getRules() {
+        final List<FuncotationFiltrationRule> lofFiltrationRules = new ArrayList<>();
+        // 1) 1) Variant classification is FRAME_SHIFT_*, NONSENSE, START_CODON_DEL, and SPLICE_SITE
+        // (within 2 bases on either side of exon or intron) on any transcript.
+        // TODO
```

Comment:
What is the TODO?  Can this be removed or addressed?

Reply:
Leftover, removed